src/modules/FSXAProxyApi.spec.ts

```diff
@@ -46,47 +46,45 @@ describe('FSXAProxyAPI', () => {
       proxyApi = new FSXAProxyApi(baseURL, LogLevel.NONE)
     })
     it('should make a fetch request to retrieve given id', async () => {
-      fetchMock.mockResponseOnce(JSON.stringify({ test: 'name' }))
+      fetchMock.mockResponseOnce(JSON.stringify({ mappedItems: [{ test: 'name' }] }))
 
-      proxyApi.fetchElement({ id, locale })
+      await proxyApi.fetchElement({ id, locale })
 
       const fetchMockBody = JSON.parse(fetchMock.mock.calls[0][1]?.body as string)
-
       const actualRequestUrl = fetchMock.mock.calls[0][0]
       const expectedRequestUrl = `${baseURL}${FSXAProxyRoutes.FETCH_ELEMENT_ROUTE}`
-
       expect(expectedRequestUrl).toEqual(actualRequestUrl)
-
       const actualRequestId = fetchMockBody.id
-
       expect(id).toEqual(actualRequestId)
-
       const actualRequestLocale = fetchMockBody.locale
-
       expect(locale).toEqual(actualRequestLocale)
     })
 
-    it('should throw an not found error when the response is 404', async () => {
+    it('should throw an not found error when the response is 404', () => {
       fetchMock.mockResponseOnce('', { status: 404 })
       const actualRequest = proxyApi.fetchElement({ id, locale })
-      expect(actualRequest).rejects.toThrow(FSXAApiErrors.NOT_FOUND)
+      return expect(actualRequest).rejects.toThrow(FSXAApiErrors.NOT_FOUND)
```

Comment:
This async was a relic from how the code was incorrectly tested previously. We don't need it anymore, since we are not awaiting anything here, but instead return a promise through the expect statement at the end. see: https://jestjs.io/docs/asynchronous